CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
-<!-- markdown-link-check-disable -->
-
 # Changelog
 
+<!-- markdown-link-check-disable -->
```

Comment:
This has to be placed under `# Changelog`, otherwise the changelog is not recognized.